src/web/Dockerfile

```diff
@@ -79,6 +80,9 @@ ENV NEXT_PUBLIC_SUPABASE_URL ${SUPABASE_URL}
 ARG ANON_KEY
 ENV NEXT_PUBLIC_ANON_KEY ${ANON_KEY}
 
+ARG DEPENDENCY_DISCOVERY_URL
+ENV NEXT_PUBLIC_DEPENDENCY_DISCOVERY_URL ${DEPENDENCY_DISCOVERY_URL}
```

Comment:
This means I need to add this to our Vercel app, which I'll do when I finish this review.

Reply:
Done.  This should work with staging on Vercel once you rebase/rebuild this PR.